src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,99 @@
+.catalog__content {
+  width: catalogSize($card-width, $card-indent);
+  display: flex;
+  margin: 0 auto;
+  justify-content: center;
+  flex-wrap: wrap;
+  padding-top: 50px;
+  padding-bottom: 73px;
+}
+
+.catalog__card {
+  margin-right: $card-indent;
+}
+
+.catalog__card:nth-child(4n) {
+  margin-right: 0;
+}
+
+.card {
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
```

Comment:
looks like `catalog `and `card `are different blocks. Move them to separate files